perfzero/lib/perfzero/utils.py

```diff
@@ -441,7 +441,7 @@ def cleanup_tpu(parameters):
   base_cmd = 'gcloud compute tpus execution-groups delete'
 
   args = [
-      '--name={}'.format(parameters.get('name')),
+      '{}'.format(parameters.get('name')),
```

Comment:
Seems this is part of base_cmd. Maybe move it there?

Reply:
parameters.get() are args on top of the static base command, so thought it would be relevant here (and consistent with the other args).